### DISCUSSION TITLE
Pnp Search Template Debug Problem

### DISCUSSION THREAD

**Discussion by iamhuman2414** [microsoft-search/pnp-modern-search#2024] [Q&A] [answered]
Hello!

Any idea why I'm seeing duplicate entries in my ReportPDFLink column when I check template debug? In my list, all records in my ReportPDFLink column show just one or single URL, however, in the template debug, I see two of the same URL's separated by comma.

This is what I see in the PnP Search Results Template Debug...
"ReportPDFLink0OWSURLH": "https://ourcompany.sharepoint.com/sites/testsite/MyList/file01.pdf, https://ourcompany.sharepoint.com/sites/testsite/MyList/file01.pdf"

Any idea why?

**Comment by iamhuman2414** [comment, marked as answer]
I figured out a workaround, I changed the type from URL to Single Line of Text, that did the trick. But would be nice to know how to do this for URL-type columns

**Reply by FranckyC** [reply]
Actually it is not a bug but the normal behavior for URL columns and xxxOWSURLH managed properties. SharePoint stores them as URL + alternative text. You see duplicates because set the same value for both in your item field value. Also to consume this type of field in templates, you must use the `getUrlField` helper. Ex: `{{getUrlField MyPropertyOWSURLH "Title"}}` or `{{getUrlField MyPropertyOWSURLH "URL"}}`. See [https://microsoft-search.github.io/pnp-modern-search/extensibility/templating/#handlebars-helpers](https://microsoft-search.github.io/pnp-modern-search/extensibility/templating/#handlebars-helpers)

**Reply by iamhuman2414** [reply]
Thanks!!!
